### PR TITLE
Fix java template for default build

### PR DESCRIPTION
**Pull request by quoctruong**
Debugging on VSCode (by running Cloud Code Deploy and then run Attach configuration), is not working for the default jib build.

### CODE REVIEW

File: java/java-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml [hunk 1]
```diff
@@ -28,3 +28,5 @@ spec:
           value: "8080"
         - name: GUESTBOOK_API_ADDR
           value: java-guestbook-backend:8080
+        - name: JAVA_TOOL_OPTIONS
```
Comment (etanshaul) [inline java/java-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml]: Is this needed for the non Skaffold debug workflow? If so, did you confirm that Skaffold debug is fine if this is already present (I’m pretty sure it is).

Comment (quoctruong) [reply java/java-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml]: Yes, I tried skaffold debug with this setting and it works as well.

**Review by simonz130 — APPROVED** [review]

File: java/java-hello-world/kubernetes-manifests/hello.deployment.yaml [hunk 2]
```diff
@@ -19,4 +19,6 @@ spec:
         - containerPort: 8080
         env:
         - name: PORT
-          value: "8080"
\ No newline at end of file
+          value: "8080"
+        - name: JAVA_TOOL_OPTIONS
```
Comment (simonz130) [inline java/java-hello-world/kubernetes-manifests/hello.deployment.yaml]: we need to add a comment that explains the purpose and what happens when folks want to remove it

**Comment by quoctruong** [comment]
@simonz130 Added the comments.

**Review by simonz130 — APPROVED** [review]

**Comment by simonz130** [comment]
Merging despite the broken CI - it fails on unrelated issue and we will work to fix that issue separately.

**Pull request merged**